test/externalTests/colony.sh

```diff
@@ -34,7 +34,7 @@ function colony_test
     FORCE_ABIv2=false
     CONFIG="truffle.js"
 
-    truffle_setup https://github.com/erak/colonyNetwork.git develop_060
+    truffle_setup https://github.com/aarlt/colonyNetwork.git natspec_on_state_variables
```

Comment:
I think this line should change to github.com/ethereum/colonyNetwork.git at some point *before* the PR merge. to not have accidental external dependencies (the old code depending on erak's github repo isn't the best option either). @chriseth, what do you think?

Reply:
We shouldn't move colonyNetwork to the ethereum namespace.

Reply:
Is this going to be adjusted once https://github.com/solidity-external-tests is done?